code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -643,44 +645,64 @@ func (c *Chain) addInitialStakes(stakes []state.InitStake, creationDate common.T
 				v.ProviderType, v.ProviderID, v.ClientID)
 		}
 
-		sp.Pools[v.ClientID] = &stakepool.DelegatePool{
+		dp := &stakepool.DelegatePool{
 			Balance:      v.Tokens,
+			Reward:       0,
+			Status:       spenum.Active,
 			DelegateID:   v.ClientID,
-			RoundCreated: 1,
+			RoundCreated: 0,
 			StakedAt:     creationDate,
 		}
 
-		edbDelegatePools = append(edbDelegatePools, &event.DelegatePool{
-			PoolID:       v.ClientID,
-			ProviderType: int(providerType),
-			ProviderID:   v.ProviderID,
-			DelegateID:   v.ClientID,
-			Balance:      v.Tokens,
-			RoundCreated: 1,
-		})
+		sp.Pools[v.ClientID] = dp
 
 		if err := sp.Save(providerType, v.ProviderID, balances); err != nil {
 			logging.Logger.Debug("init stake - save staking pool failed", zap.Error(err))
 			return err
 		}
 
-		logging.Logger.Info("init stake", zap.String("sc ID", v.ProviderID), zap.Any("tokens", v.Tokens))
-	}
-
-	if c.EventDb == nil {
-		return nil
-	}
+		if c.EventDb == nil {
+			continue
+		}
 
-	if len(edbDelegatePools) == 0 {
-		return nil
-	}
+		log.Println("=============> start to save addInitialStakes events")
```

Comment:
Use `logging.Logger` and keep the logging message consistent, we don't use `=====>` in 0chain repo

Reply:
Please ignore any log starting with `=====>`, I use them to easily find active logs and they are also easy to remove because of the prefix. I will remove them when the PR is ready.